lib/influxdb2/client/flux_csv_parser.rb

```diff
@@ -187,6 +187,12 @@ def _add_column_names_and_tags(table, csv)
         column.label = csv[i]
         i += 1
       end
+
+      duplicates = table.columns.group_by { :label }.select { |_k, v| v.size > 1 }
+
+      warning = "The response contains columns with duplicated names: #{duplicates.keys.join(', ')}\nYou should use the
+ 'FluxRecord.row to access your data instead of 'FluxRecord.values' hash."
```

Comment:
The warning massage is not properly formatted:

![image](https://user-images.githubusercontent.com/455137/194517642-a1a8c2eb-8226-4098-b2aa-58ddefd700ff.png)


```suggestion
      warning = "The response contains columns with duplicated names: #{duplicates.keys.join(', ')}.
You should use the 'FluxRecord.row' to access your data instead of 'FluxRecord.values' hash."
```